### PR TITLE
Applied Gitflow

### PR DESCRIPTION
**Pull request by AsimKhan2019**
Implemented the Book store structure using the components to set up the project. Created the book list, book, category, and navigation component. Applied the routing function amongst books and categories. Styling is not applied to the project as it is not required now. Followed the Git flow approach to the project.

### CODE REVIEW

File: src/components/Categories.js [hunk 1]
```diff
@@ -2,7 +2,9 @@ import React from 'react';
 
 const Categories = () => (
   <div className="category">
-    <button type="button">Check Status</button>
+    <button type="button">
+      Check Status
+    </button>
```
Comment (codecaiine) [inline src/components/Categories.js:7]: - [x] Good job so far with your project :+1:  But there is an issue with your form :cry: From the project requirement  you need to render a form with inputs for the title and the author of the book, you missed to add your input inside your form tag to follow best practice Please fix it :smile: 

#### AddBook.js  screenshot
Screenshot from 2022-08-11 17-33-30

- [x]  As a requirement please add a button for removing books :smile: 

#### Books.js screenshot
Screenshot from 2022-08-11 17-35-39

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @AsimKhan2019 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good project implementation :heavy_check_mark: 
- Good use of git-flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [x] Please I would like to recommend making your PR title reflect what you are doing in your project . 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Categories.js (hunk 1 above)
Comment (AsimKhan2019) [reply src/components/Categories.js:7]: @codecaiine Thank you for the review, I will try to fix the issues as soon as possible.

Comment (Whoistolu) [reply src/components/Categories.js:7]: Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @AsimKhan2019 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AsimKhan2019** [comment]
@Whoistolu Thank you for the approval

**Pull request merged**